jekyll/_cci2/api-developers-guide.md

```diff
@@ -450,6 +450,184 @@ The following section details the steps you need to follow to download artifacts
+
+```
+json
+{
+	"next_page_token": null,
+	"items": [{
+		"name": "build",
+		"metrics": {
+			"success_rate": 0.5975609756097561,
+			"total_runs": 82,
+			"failed_runs": 33,
+			"successful_runs": 49,
+			"throughput": 11.714285714285714,
+			"mttr": 46466,
+			"duration_metrics": {
+				"min": 8796,
+				"max": 20707,
+				"median": 11656,
+				"mean": 12847,
+				"p95": 18856,
+				"standard_deviation": 3489.0
+			},
+			"total_credits_used": 16216608
+		},
+		"window_start": "2020-01-15T03:20:24.927Z",
+		"window_end": "2020-01-21T23:23:04.390Z"
+	}, {
+		"name": "docker_build",
+		"metrics": {
+			"success_rate": 1.0,
+			"total_runs": 1,
+			"failed_runs": 0,
+			"successful_runs": 1,
+			"throughput": 1.0,
+			"mttr": 0,
+			"duration_metrics": {
+				"min": 1570,
+				"max": 1570,
+				"median": 1570,
+				"mean": 1570,
+				"p95": 1570,
+				"standard_deviation": 0.0
+			},
+			"total_credits_used": 5154
+		},
+		"window_start": "2020-01-19T15:00:16.032Z",
+		"window_end": "2020-01-19T15:26:26.648Z"
+	}, {
+		"name": "ecr_gc",
+		"metrics": {
+			"success_rate": 1.0,
+			"total_runs": 167,
+			"failed_runs": 0,
+			"successful_runs": 167,
+			"throughput": 23.857142857142858,
+			"mttr": 0,
+			"duration_metrics": {
+				"min": 31,
+				"max": 96,
+				"median": 46,
+				"mean": 49,
+				"p95": 72,
+				"standard_deviation": 11.0
+			},
+			"total_credits_used": 3482
+		},
+		"window_start": "2020-01-15T01:45:03.613Z",
+		"window_end": "2020-01-21T23:46:25.970Z"
+	}]
+}
+```
+
+Notice that in this JSON response, you will receive detailed metrics for the set of workflows that were run, including:
+
+- `success_rate` - the rate of success
+- `total_runs` - the total number of runs that were performed
+- `failed_runs` - the number of runs that failed
+- `successful_runs` - the number of runs that were successful
+- `throughput` - 
+- `mttr` - 
+- `duration_metrics` - an array of specific metrics that detail the duration of the workflow, which includes `min`, `max`, `median`, `mean`, `p95`, and `standard_deviation`
+- `total credits used` - the total number of credits used
+- `windows_start & windows_end` - the time the window was first started, and then ended
+
+**Note** The above example only shows just a few builds. When you run this command, you may receive up to 250 individual builds that you can review in much more detail.
+
+### Reviewing Individual Build Metrics
+
+Now that you have retrieved aggregated data for up to 250 different builds, you will most likely want to review specific information about a single build, or smaller number of builds, to ensure that your builds are running efficiently. To review an individual build, follow the steps below.
+
+1. Using your project-slug from the previous API call you made to return workflow data, make a GET API call to the following insights endpoint:
+
+    ```sh
+    curl -X GET https://circleci.com/api/v2/insights/{project-slug}/workflows/builds
+    -H 'Accept: application/json'
```

Comment:
same as above - `--header` to be consistent, and need to add CIRCLE_TOKEN

Reply:
Fixed